CheckerPrinter/src/checkerprinter/InMemory.java

```diff
@@ -92,13 +92,19 @@ protected boolean initCheckerArgs(JsonObject optionsObject) {
             this.exceptionMsg = "Error: Cannot find indicated checker.";
             return false;
         }
-        this.checkerOptionsList = Arrays.asList( "-Xbootclasspath/p:" +
-                this.CHECKER_FRAMEWORK + "/checker/dist/jdk8.jar",
-                "-processor",
-                checker);
+        this.checkerOptionsList = new ArrayList<String>();
+
+        this.checkerOptionsList.add("-Xbootclasspath/p:" + this.CHECKER_FRAMEWORK + "/checker/dist/jdk8.jar");
+        this.checkerOptionsList.add("-processor");
+        this.checkerOptionsList.add(checker);
+
         if (optionsObject.getBoolean("has_cfg")) {
-            // String cfgLevel = optionsObject.getString("cfg_level");
-            // TODO: add CFG Visualization
+
+            //assume exists CFG directory
+            this.checkerOptionsList.add("-classpath");
+            this.checkerOptionsList.add(this.CHECKER_FRAMEWORK + "/checker/dist/checker.jar");
```

Comment:
Isn't it a bit surprising that the `checker.jar` file is only needed when that flag is given? You also need it when normally running the processor. If this is indeed needed, can you add a comment documenting why?